pym/bob/input.py

```diff
@@ -1559,6 +1567,7 @@ def __init__(self, recipe, tools, diffTools, sandbox, diffSandbox,
         self.states = states
         self.pkgId = pkgId
         self.fingerprintMask = fingerprintMask
+        self.jobServer = useJobServer
```

Comment:
There is no need to store the value in the package because it will always be the same as the recipe value. Just access `self.recipe...` to get the value.